test/suites/server/controllers/shantytownCommentController/create.spec.js

```diff
@@ -0,0 +1,183 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const { mockReq, mockRes } = require('sinon-express-mock');
+const { serialized: fakeUser } = require('#test/utils/user');
+
+const { expect } = chai;
+chai.use(sinonChai);
+
+const { factory } = require('#server/controllers/shantytownCommentController/create');
+const ServiceError = require('#server/errors/ServiceError');
+
+describe.only('controllers/shantytownComment', () => {
+    describe('create()', () => {
+        it('fait appel au service shantytownComment/createComment() pour insérer le commentaire', async () => {
+            const srvCreateComment = sinon.stub();
+            const user = fakeUser();
+            const shantytown = { id: 1 };
+            const ctlCreate = factory({ createComment: srvCreateComment });
```

Comment:
Okay, je crois mieux comprendre pourquoi ces factories 😅  

Je pense pas que t'aies besoin de faire ça (d'utiliser des factories & de passer des services en paramètres) pour tester ton controller. 

- Mettre ta couche logique dans des services tiers (création de commentaire, call alerte slack) et utiliser des simple require dans ton controller
- Mocker ces services tiers : https://sinonjs.org/how-to/stub-dependency/